### PR TITLE
Adding wiki for browser based ACL calls. 

### PR DESCRIPTION
**Pull request by uditsaxena**
I added files to demonstrate ACL calls from the browser. 

This includes a new example file similar to ClientExample, and an examples/acl folder in the root directory. 

The wiki to be included is [here](https://github.com/uditsaxena/GNS/wiki/Making-an-ACL-Call-from-the-browser)

### CODE REVIEW

**Comment by westyUMass** [comment]
It looks like there is an issue with the SelectTest that didn't appear during previous pull request testing. The client command(s) that are breaking are scheduled to be deprecated and those tests will be removed. Try rerunning the travis check in the meantime.

**Comment by kanantharamu** [comment]
Udit, the CI test for this pull request has failed. Can you take a look into the [build log](https://travis-ci.org/MobilityFirst/GNS/builds/222154601) to see if the changes you made are causing this? If so, please fix it and resubmit a pull request that passes the CI test.

**Comment by westyUMass** [comment]
@uditsaxena, See my comment above about SelectTest and note that I fixed that issue in the master branch. I would merge the latest master into your branch and try creating a new pull request.

**Comment by uditsaxena** [comment]
Yes, @westyUMass @kanantharamu I am going to look into this. Apologies for the delay and thanks !

**Comment by uditsaxena** [comment]
@westyUMass @kanantharamu I have updated the PR.

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
 .DS_Store
+.idea/
```
Comment (uditsaxena) [inline .gitignore]: I think we can leave out the gitignore file altogether. I will update the PR now.

**Pull request merged**